backend/kesaseteli/applications/jobs/daily/clean_rejected_youth_applications.py

```diff
@@ -0,0 +1,17 @@
+from dateutil.relativedelta import relativedelta
+from django.utils import timezone
+from django_extensions.management.jobs import DailyJob
+
+from applications.models import YouthApplication
+
+
+class Job(DailyJob):
+    help = "Clean all rejected youth applications older than one year"
+
+    def execute(self):
+        one_year_from_now = timezone.now() - relativedelta(years=1)
+        YouthApplication.objects.filter(
+            created_at__lte=one_year_from_now,
+            handled_at__isnull=False,
+            status="rejected",
```

Comment:
The enum value could be imported and used through the enum rather than as a string literal i.e.
```python
from applications.enums import YouthApplicationStatus
...
status=YouthApplicationStatus.REJECTED
```

Reply:
fixed

Reply:
fixed